pkg/verifier/gcp/entry_point.go

```diff
@@ -25,25 +26,32 @@ func (g *GcpVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		vei.InstanceType = "e2-standard-2"
 	}
 
+	vei.InstanceType = "e2-micro"
```

Comment:
can we make `e2-micro` a constant here? similar to how we do for `aws/entry_point.go#L23`. That way we can reuse it everywhere (including tests) and only need to change it in one spot if we have to update this in the future.